# 2022-01-20 07:03:26.072086: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 AVX512F FMATo enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.2022-01-20 07:03:26.111593: I tensorflow/core/common_runtime/process_util.cc:146] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.Traceback (most recent call last):  File "/home/neo/mlfuzz/tensorflow/synthesized/all-crashes/all/AddManySparseToTensorsMap_21718fa64d31d11ae37f52f756ad50c8.py", line 6, in <module>    tf.raw_ops.AddManySparseToTensorsMap(sparse_indices=arg_0, sparse_values=arg_1, sparse_shape=arg_2)  File "/home/neo/anaconda3/envs/tf/lib/python3.9/site-packages/tensorflow/python/util/tf_export.py", line 404, in wrapper    return f(**kwargs)  File "/home/neo/anaconda3/envs/tf/lib/python3.9/site-packages/tensorflow/python/ops/gen_sparse_ops.py", line 79, in add_many_sparse_to_tensors_map    _ops.raise_from_not_ok_status(e, name)  File "/home/neo/anaconda3/envs/tf/lib/python3.9/site-packages/tensorflow/python/framework/ops.py", line 6941, in raise_from_not_ok_status    six.raise_from(core._status_to_exception(e.code, message), None)  File "<string>", line 3, in raise_fromtensorflow.python.framework.errors_impl.InternalError: Encountered overflow from large input shape. [Op:AddManySparseToTensorsMap]

import tensorflow as tf

arg_0 = tf.constant(0, shape=[3,2], dtype=tf.int64)
arg_1 = tf.constant("a", shape=[3], dtype=tf.string)
arg_2 = tf.constant(-1879048192, shape=[2], dtype=tf.int64)
tf.raw_ops.AddManySparseToTensorsMap(sparse_indices=arg_0, sparse_values=arg_1, sparse_shape=arg_2)